web/src/pages/billing/BillingCostByMonth.tsx

```diff
@@ -0,0 +1,239 @@
+import * as React from 'react'
+import { useLocation, useNavigate, useSearchParams } from 'react-router-dom'
+import { Button, Card, Grid, Input, Message, Table as SUITable } from 'semantic-ui-react'
+import {
+    BillingApi,
+    BillingColumn,
+    BillingSource,
+    BillingTotalCostQueryModel,
+    BillingTotalCostRecord,
+} from '../../sm-api'
+
+import { getMonthStartDate, getMonthEndDate } from '../../shared/utilities/monthStartEndDate'
+import { IStackedAreaByDateChartData } from '../../shared/components/Graphs/StackedAreaByDateChart'
+import BillingCostByMonthTable from './components/BillingCostByMonthTable'
+import LoadingDucks from '../../shared/components/LoadingDucks/LoadingDucks'
+import generateUrl from '../../shared/utilities/generateUrl'
+
+const BillingCostByTime: React.FunctionComponent = () => {
+    const [searchParams] = useSearchParams()
+
+    const [start, setStart] = React.useState<string>(
+        searchParams.get('start') ?? getMonthStartDate()
```

Comment:
Can this be start of year?

Reply:
Changed that to fixed 202201 - > 202301 as per slack chat.